Simplify schedules page (#14315)
PrefectHQ/prefect: docs/3.0rc/automate/add-schedules.mdx

You can add one or more schedules to any [served or deployed](/3.0rc/deploy/serve-flows/) flow.
Schedules tell Prefect when and how to create new flow runs.
You can add a schedule to a deployed flow in the Prefect UI, in the CLI through the `prefect deployment schedule` command, or the `prefect.yaml` configuration file.

## Create schedules

There are several ways to create a schedule for a deployment:

- Through the Prefect UI
- With the `cron`, `interval`, or `rrule` parameters if building your deployment with the 
[`serve` method](/3.0rc/develop/write-flows/#serving-a-flow) of the `Flow` object or 
[the `serve` utility](/3.0rc/develop/write-flows/#serving-multiple-flows-at-once) for managing multiple flows simultaneously
- If using [worker-based deployments](/3.0rc/deploy/control-runs/workers/)
  - When you define a deployment with `flow.serve` or `flow.deploy`
  - Through the interactive `prefect deploy` command
  - With the `deployments` -> `schedules` section of the `prefect.yaml` file

### Create schedules in the UI

You can add schedules in the **Schedules** section of a **Deployment** page in the UI.
To add a schedule select the **+ Schedule** button. 

> A modal dialog will open. 

Reviewer: ```suggestion
```

Author: Agreed. Removed

Choose **Interval** or **Cron** to create a schedule.

<Note>
**What about RRule?**
The UI does not support creating RRule schedules. 
However, the UI will display RRule schedules that you've created through the command line.
</Note>

The new schedule appears on the **Deployment** page where you created it. 
New scheduled flow runs are visible in the **Upcoming** tab of the **Deployment** page.

> To edit an existing schedule, select the **Edit** from the three-dot menu next to a schedule on a **Deployment** page.

Reviewer: ```suggestion
To edit an existing schedule, select **Edit** from the three-dot menu next to a schedule on a **Deployment** page.
```

> See the [Settings docs](/3.0rc/manage/configure-client/) for more information on altering your settings.

Reviewer: ```suggestion
See the [Settings docs](/3.0rc/manage/settings-and-profiles/) for more information on altering your settings.
```
